src/scales/scale.time.js

```diff
@@ -146,16 +146,26 @@ module.exports = function(Chart) {
 			minUnit: 'millisecond',
 
 			// defaults to unit's corresponding unitFormat below or override using pattern string from http://momentjs.com/docs/#/displaying/format/
+			// Use more detailed date strings on major ticks and summary info on minor ticks
 			displayFormats: {
-				millisecond: 'h:mm:ss.SSS a', // 11:20:01.123 AM,
-				second: 'h:mm:ss a', // 11:20:01 AM
-				minute: 'h:mm a', // 11:20 AM
+				millisecond: '.SSS', // .123,
```

Comment:
It was changed here: https://github.com/chartjs/Chart.js/commit/2d7c1f0d2c330e4b201c6fdcdd0489a1f1dbda35

It's breaking in the sense that the chart will be displayed different, but nearly every PR changes the way a chart is displayed. It's not breaking in the sense that no APIs have changed, no one's chart will stop displaying, etc.